api/classes/prerequisites.js

```diff
@@ -85,7 +85,7 @@ class Prerequisites {
         const usernameParam = request.auth.credentials.username;
 
         const result = Promise.fromCallback(next => {
-          return request.server.methods.cache.user(usernameParam, next);
```

Comment:
It was just the method name, it didn't actually represent cache. I removed it because otherwise the method to drop a key would be `request.server.methods.cache.user.cache.drop`. It seemed redundant so I remover the first `cache`. It still caches the returned value of the method since we have a cache config setup for the method